From 9ec8da54d52fa1cf94b790ed0daaae0aa26c1e4e Mon Sep 17 00:00:00 2001
Message-Id: <9ec8da54d52fa1cf94b790ed0daaae0aa26c1e4e.1374142811.git.mkletzan@redhat.com>
In-Reply-To: <cover.1374142811.git.mkletzan@redhat.com>
References: <cover.1374142811.git.mkletzan@redhat.com>
From: Marc Deslauriers <marc.deslauriers@canonical.com>
Date: Mon, 23 Apr 2012 09:57:15 -0400
Subject: [RHEL-6.5 python-virtinst PATCH v2 6/7] virtinst: Add latest Ubuntu
 releases

https://bugzilla.redhat.com/show_bug.cgi?id=958496

Hi,

Here's a patch to add the latest Ubuntu releases, to add "LTS" when
appropriate, and to mark Maverick as being unsupported.

Thanks,

Marc.

>From 287d13ee843bcbc33365075573708e45a2791329 Mon Sep 17 00:00:00 2001
From: Marc Deslauriers <marc.deslauriers@ubuntu.com>
Date: Mon, 23 Apr 2012 09:54:32 -0400
Subject: [PATCH] Add latest Ubuntu releases, and add LTS when appropriate.

(cherry picked from commit f9ed4e76821521e3a6de0477d20dddf2987af4a5)

Conflicts:
	man/en/virt-install.1 -- autogenerated downstream
---
 virtinst/osdict.py | 21 +++++++++++++++++++--
 1 file changed, 19 insertions(+), 2 deletions(-)

diff --git a/virtinst/osdict.py b/virtinst/osdict.py
index a0a9414..7fd9a1c 100644
--- a/virtinst/osdict.py
+++ b/virtinst/osdict.py
@@ -526,7 +526,7 @@ OS_TYPES = {
         },
     },
     "ubuntulucid": {
-        "label": "Ubuntu 10.04 (Lucid Lynx)",
+        "label": "Ubuntu 10.04 LTS (Lucid Lynx)",
         "distro": "ubuntu",
         "supported": True,
         "devices" : {
@@ -537,7 +537,6 @@ OS_TYPES = {
     "ubuntumaverick": {
         "label": "Ubuntu 10.10 (Maverick Meerkat)",
         "distro": "ubuntu",
-        "supported": True,
         "devices" : {
             DISK : VIRTIO_DISK,
             NET  : VIRTIO_NET,
@@ -561,6 +560,24 @@ OS_TYPES = {
             NET  : VIRTIO_NET,
         },
     },
+    "ubuntuprecise": {
+        "label": "Ubuntu 12.04 LTS (Precise Pangolin)",
+        "distro": "ubuntu",
+        "supported": True,
+        "devices" : {
+            DISK : VIRTIO_DISK,
+            NET  : VIRTIO_NET,
+        },
+    },
+    "ubuntuquantal": {
+        "label": "Ubuntu 12.10 (Quantal Quetzal)",
+        "distro": "ubuntu",
+        "supported": True,
+        "devices" : {
+            DISK : VIRTIO_DISK,
+            NET  : VIRTIO_NET,
+        },
+    },

     "generic24": {
         "label": "Generic 2.4.x kernel"
-- 
1.8.3.2

